### ISSUE TITLE
Repository goals and format

### ISSUE THREAD

**Issue by Bogidon** [EastRobotics/GeneralRoboticsLibrary#2]
Picking up discussion from #1.

There's lots lots lots of great advice on the [VEX forum](http://vexforum.com) (@raysarebest, I haven't been as active on the ROBOTC forum). [JPearman](http://www.vexforum.com/index.php/member/activity/3759-jpearman) in particular has been a great boon to the greater robotics community. One thing that I find thoroughly lacking is a comprehensive reference for ROBOTC. The wiki is lacking, the forums are difficult to navigate, and personal sites (like [Bots 'n' Stuff](http://botsnstuff.com)) don't get a lot of active development (and are sometimes close sourced!!!).

What I think would be great is a wiki-type repository that contains consumable pieces of information. My first intuition is to suggest writing the "pages" in Markdown. This way we can include code, syntax highlighting and all, and the format is easy to work with over Github. Check out the repo [Hack Club](https://github.com/hackclub/hackclub/tree/master/workshops) for a great example.

Mainly though I think we should be careful to not rewrite much information that is available in other places. Maybe the first step of adding documentation for a new "component" should be researching what others have written. Some solutions are more efficient computationally and mechanically than others.

@raysarebest @jcgrif

**Comment by jcgrif** [comment]
I like the idea of a wiki-style repository.  The purpose is not to redo anything that anyone else has done, but to be creative and share code concepts that have worked, or when proof-of-concepts are built, place the code here, so that some future student or even an current student on another CHE team that has a similar idea can leverage the work that has already been done or collaborate on something.  I am a firm believer in building proof of concept platforms and then modularizing and integrating if possible.

**Comment by raysarebest** [comment]
Agreed. That being said, we can put the example code in the repository, and then use GitHub's built-in Wiki feature, which tbh I've been looking for an excuse to try out forever
